crates/router/src/core/payment_methods.rs

```diff
@@ -305,11 +305,21 @@ pub async fn render_pm_collect_link(
                         .attach_printable("Failed to serialize PaymentMethodCollectLinkDetails")?
                 );
 
+                let document =
+                    include_str!("./generic_link/payment_method_collect/initiate/index.html");
+                let scripts =
+                    include_str!("./generic_link/payment_method_collect/initiate/script.js");
+                let styles =
+                    include_str!("./generic_link/payment_method_collect/initiate/styles.css");
```

Comment:
we can do this `api.rs` right! since all links follow the same convension. we shouldn't importing here